src/core_plugins/elasticsearch/lib/health_check.js

```diff
@@ -1,18 +1,12 @@
 import Promise from 'bluebird';
 import elasticsearch from 'elasticsearch';
-import createKibanaIndex from './create_kibana_index';
 import kibanaVersion from './kibana_version';
 import { ensureEsVersion } from './ensure_es_version';
 import { ensureNotTribe } from './ensure_not_tribe';
 import { patchKibanaIndex } from './patch_kibana_index';
+import { format } from 'util';
```

Comment:
__optional nit:__ I know it was here before (hence up to you), but it seems the only place it's used in is:
```js
plugin.status.red(format('Unable to connect to Elasticsearch at %s.', url));
```
that is basically just 
```js
plugin.status.red(`Unable to connect to Elasticsearch at ${url}`);
```